benchmarks/7.2.0/7.2_performance_summary.md

```
@@ -275,28 +275,30 @@ Concurrent Users | Single Node 4 Core | Two Node 2 Core | Two Node 4 Core | Thre
 
 Performance Comparison of Different Node Configurations with 95th Percentile of Response Time (ms)
 
+> **Note:** The 50 concurrent-user results for the Two Node 4 Core, Three Node 4 Core and Four Node 4 Core configurations, and the 100 concurrent-user result for the Four Node 4 Core configuration, have been updated as per our latest performance run.
+
 Concurrent Users | Single Node 4 Core | Two Node 2 Core | Two Node 4 Core | Three Node 4 Core | Four Node 4 Core
```

markdownlint-cli2 (0.23.0)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)